tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,104 @@
+    brown_squad,
+    skipif_ocs_version,
+    bugzilla,
+)
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier2,
+    skipif_external_mode,
+    cloud_platform_required,
+)
+from ocs_ci.helpers.e2e_helpers import run_metadata_io_with_cephfs
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2141422")
+@brown_squad
+@skipif_ocs_version("<4.15")
+@skipif_external_mode
+@cloud_platform_required
+class TestMdsCacheTrimStandby(E2ETest):
+    @pytest.fixture(scope="function", autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            """
+            This function will call a function to clear the mds memory usage gradually
+
+            """
+            cluster.bring_down_mds_memory_usage_gradually()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.polarion_id("OCS-6280")
+    def test_mds_cache_trim_on_standby_replay(self, dc_pod_factory):
+        """
+        Verifies whether the MDS cache is trimmed or not in standby-replay mode.
+
+        Steps:
+        1. Run metadata IO on at least 5 pods, so that the memory consumption will be rapid and high.
+        2. When mds memory utilisation reached 75% [it is similar to 150% of cache utilisation], there is a possibility
+        to trigger MDS cache oversized warning.
+        3. Now, the script look for such warnings in ceph health.
+        4. If found the warning, it identifies whether the warning is from Active or standby mds
+         by searching the warning in the mds pod logs.
+        5. After that, it identifies whether the cache trim is happening or not on stand by MDS.
+        6. Test case passes if MDS cache oversized warning not found in Standby mds and
+          if cache trim is happening in Standby mds pod.
+
+        """
+        run_metadata_io_with_cephfs(dc_pod_factory, no_of_io_pods=5)
+        log.info(
+            "Starting metadata IO in the background. Monitoring for MDS cache alerts."
+        )
+
+        trim_msgs = ["cache trim"]
+        cache_warning = "MDSs report oversized cache"
+
+        for sampler in TimeoutSampler(
+            timeout=1800,
+            sleep=20,
+            func=cluster.get_active_mds_memory_utilisation_in_percentage,
+        ):
+            if sampler > 75:
+                break
+            else:
+                log.warning("MDS memory consumption is not yet reached target")
+
+        active_mds_mem_util = cluster.get_active_mds_memory_utilisation_in_percentage()
+        sr_mds_mem_util = (
+            cluster.get_standby_replay_mds_memory_utilisation_in_percentage()
+        )
+
+        log.info(f"Active MDS memory utilization: {active_mds_mem_util}%")
+        log.info(f"Standby-replay MDS memory utilization: {sr_mds_mem_util}%")
+        ceph_health_detail = cluster.ceph_health_detail()
```

Comment:
you can remove from this line and add it before checking the warning in health detail output line 99